dotnet/QnAMaker/SDK-based-quickstart/Program.cs

```diff
@@ -239,6 +237,49 @@ private static async Task GenerateAnswer(IQnAMakerRuntimeClient runtimeClient, s
         }
         // </GenerateAnswer>
 
+        // <GenerateAnswer>
```

Comment:
This snippet should be named "GenerateAnswerPreview" to avoid name collision.

Reply:
okay. Will do in next PR.